Packs/EWS/Integrations/EWSv2/EWSv2.py

```diff
@@ -1204,6 +1204,16 @@ def parse_incident_from_item(item, is_fetch):
             except ErrorIrresolvableConflict:
                 time.sleep(0.5)
                 item.save()
+            except ValueError as e:
+                if len(item.subject) > 255:
```

Comment:
Can item.subject be None?
If so, I suggest 
```suggestion
                if item.subject and len(item.subject) > 255:
```